README.md

```diff
@@ -293,6 +294,21 @@ strategies:
     enabled: true
 ````
 
+### RemovePodsViolatingTopologySpreadConstraint
+
+This strategy makes sure that pods violating [topology spread constraints](https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/)
+are evicted from nodes. This strategy requires k8s version 1.18 at a minimum. To disable this strategy, the
```

Comment:
1.18? I thought this feature was introduced in 1.16?

Reply:
It is beta in 1.18 and enabled by default. It is GA in 1.19. Prior to 1.18 it was alpha and required a feature gate to enable it.